app/config/views.yml

```diff
@@ -10,6 +10,9 @@ ezpublish:
                     home:
                         controller: ez_query:LocationQueryAction
                         params:
+                            omitted_location_ids:
```

Comment:
Yes, but this will require more work. Let's leave it for now and improve it later.